asciidoc_only/preface.asciidoc

```diff
@@ -71,7 +71,7 @@ Please address comments and questions concerning this book to the publisher:
 </ul>
 ++++
 
-We have a web page for this book, where we list errata, examples, and any additional information. You can access this page at link:$$https://www.oreilly.com/catalog/catalog_page$$[].
+We have a web page for this book, where we list errata and any additional information. You can access this page at link:$$https://www.oreilly.com/catalog/catalog_page$$[].

```

Comment:
```suggestion
We have a web page for this book, where we list errata and any additional information. You can access this page at https://www.oreilly.com/catalog/catalog_page[].
```
...just since we're in here, may as well modernize the markup? I won't make this suggestion everywhere, but, you know, it applies.